samples/nrf9160/gps/src/supl_support.c

```diff
@@ -36,10 +37,19 @@ int open_supl_socket(void)
 		.ai_canonname = NULL,
 	};
 
-	err = getaddrinfo(SUPL_SERVER, NULL, &hints, &info);
+	/* Try getaddrinfo many times, sleep a bit in between */
+	while ((err != 0) && (gai_cnt < 5)) {
+		err = getaddrinfo(SUPL_SERVER, NULL, &hints, &info);
+		if (err) {
+			printk("Failed to resolve hostname %s on IPv4, errno: %d)\n",
+				SUPL_SERVER, errno);
+			k_sleep(K_MSEC(2000) * gai_cnt);
```

Comment:
We should not spend to much time here, 2 second seems a bit to much. I worst case this may end up with 2sec + 4sec + 6sec + 8sec = 20sec. Then we have lost all of the speed up this can give compared to downloading the data from the satellites, and that is before we start the SUPL session. I think setting the timeout to 1sec and then try 3 times would suffice. That is one try with no sleep, and two with sleep (1 sec and 2 sec).

Reply:
Ok, I will change accordingly